src/features/events_list/atoms/eventListResource.ts

```diff
@@ -1,18 +1,37 @@
-import { createResourceAtom } from '~utils/atoms';
+import { createResourceAtom, combineAtoms } from '~utils/atoms';
 import { apiClient } from '~core/apiClientInstance';
 import { autoRefreshService } from '~core/autoRefreshServiceInstance';
 import { currentEventFeedAtom } from '~core/shared_state';
 import type { Event } from '~core/types';
+import { eventListFilters } from './eventListFilters';
```

Comment:
it looks like eslint fix not working 🤷‍♂️

Reply:
hmm, it's more look like lint-staged problem.
After I run `npx eslint --fix ./src` it was fixed